packages/components/package.json

```diff
@@ -89,26 +89,26 @@
     "babel-plugin-ember-template-compilation": "^2.2.4",
     "concurrently": "^8.2.2",
     "ember-basic-dropdown": "^8.1.0",
-    "ember-source": "~5.9.0",
+    "ember-source": "~5.12.0",
```

Comment:
since it's already in `peerDeps` you do not need it as a `devDeps`

Reply:
thinking out loud, but it seems likely this is leftover from when there was an actual dummy app inside this package. now that it's moved elsewhere, there's no need to specify a specific ember version?

Reply:
correct, this line should be moved to `peerDeps` and provide wide ranges that are supported, so that types can be resolved

Reply:
Removed this from `devDependencies` and left the `peerDependencies` range as it is currently as `"ember-source": "^3.28.0 || ^4.0.0 || ^5.3.0"`.